pkgs/development/compilers/rust/rustc.nix

```diff
@@ -146,14 +146,16 @@ in stdenv.mkDerivation (finalAttrs: {
     # Since fastCross only builds std, it doesn't make sense (and
     # doesn't work) to build a linker.
     "--disable-llvm-bitcode-linker"
-  ] ++ optionals (stdenv.isLinux && !stdenv.targetPlatform.isRedox) [
+  ] ++ optionals (stdenv.isLinux && !stdenv.targetPlatform.isRedox && !stdenv.targetPlatform.isWasi) [
```

Comment:
Can be dropped after https://github.com/NixOS/nixpkgs/pull/343059

Reply:
It's been merged btw

Reply:
Thank you for letting me know, but I'm a bit at a loss with the changes to `rust/lib/default.nix`. (I have an idea, but it'll require mass rebuilds. :/)